frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -357,7 +357,7 @@ const SECTIONS = updateSectionsWithPlugins({
       },
       {
         key: "enable-embedding",
-        display_name: t`Enable Embedding Metabase in other Applications`,
+        display_name: t`Embedding`,
```

Comment:
Fix setting header to match [the design](https://www.figma.com/file/SAi6pRHuhrWeVRuOHH4evg/Better-communicate-embedding-options-in-Admin-settings).
#### Before
![image](https://user-images.githubusercontent.com/1937582/177190211-b65a80b0-2528-4c76-a8c5-b62b6e9c3018.png)

#### After
![image](https://user-images.githubusercontent.com/1937582/177190309-45ea38d1-eaf0-465e-80d6-2a29a6bc1b67.png)